api-docs.md

```diff
@@ -563,6 +564,42 @@ There are, however, a few catches to this:
 - Sorting is only applied at the Druid level. Therefore, the results of a sort are not guaranteed to be accurate if Fili 
 performs any post-Druid calculations on one of the metrics that you are sorting on.
 
+### TopN ###
+
+Suppose we would like to know which three pages have the top three pageview counts for each week between January and 
+September 2014. We can easily answer such a question with a `topN` query. `topN` queries allow us to ask for the top N 
+results for each time bucket in a request. Of course, a `topN` query implies that some sort of ordering has been imposed
```

Comment:
It doesn't seem that repetitive to me, but it did give me pause when I read it the 1st time due to the seeming complexity of the sentence construction. I wouldn't be opposed to a re-work, but it wouldn't be a blocker for me either.